### ISSUE TITLE
Unable not edit a long post and access to the beginning

### ISSUE THREAD

**Issue by foxmask** [BDX-town/Mangane#317]
When I want to edit a post of 3841 cars. I am unable to edit the begining of the text

I use firefox on Linux and Android but I could not edit the beginning. When I try to access of the beginning by pulling the display, the page reload the posts of my page

I had to use fedilab to be able to change it easily

**Comment by Dehelssey** [comment]
yeah, @Cl0v1s is aware of this issue, it's because the compose box is a modal and when text is too long you can't scroll back to top...

He's reworking it to be a full-size window (like he did with emoji panel) so we don't have that problem anymore

**Comment by Cl0v1s** [comment]
This should be fixed now since editing is now made on a full page
